### ISSUE TITLE
0001078:
email features

### ISSUE THREAD

**Issue by Gloirin** [Gloirin/m2gTest#532]
**Reported by pschuele on 12 Jun 2009 11:03**

nice to have features:
see children

**Comment by Gloirin** [comment]
Depends on #745

**Comment by Gloirin** [comment]
Depends on #1682

**Comment by Gloirin** [comment]
Depends on #1679

**Comment by Gloirin** [comment]
Depends on #655

**Comment by Gloirin** [comment]
Depends on #708

**Comment by Gloirin** [comment]
Depends on #726

**Comment by Gloirin** [comment]
Depends on #1693

**Comment by Gloirin** [comment]
Depends on #745

**Comment by Gloirin** [comment]
Depends on #1682

**Comment by Gloirin** [comment]
Depends on #1679

**Comment by Gloirin** [comment]
Depends on #655

**Comment by Gloirin** [comment]
Depends on #708

**Comment by Gloirin** [comment]
Depends on #709

**Comment by Gloirin** [comment]
Depends on #726

**Comment by Gloirin** [comment]
Depends on #1693

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 1 Sep 2009 12:09**

resolved all children
